Sources/View/ReusableView.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-public protocol ReusableView: View {
```

Comment:
If we remove this conformance on `ReusableView`, I think we should add it to `ReusableViewModelView`